asmcomp/emitaux.ml

```diff
@@ -36,12 +36,16 @@ let emit_printf fmt =
 let emit_int32 n = emit_printf "0x%lx" n
 
 let emit_symbol s =
+  let symbol_separator = Compilenv.symbol_separator in
```

Comment:
You don't really need this `let` any longer.

Reply:
Sure, done.